recipes/glew/run_test.bat

```diff
@@ -0,0 +1,7 @@
+conda env create -f %RECIPE_DIR%\test\test-environment-windows.yml
```

Comment:
I completely agree that this is an ugly hack. I've subscribed to the [issue](https://github.com/conda/conda-build/issues/1059) so that whenever it is fixed I can remove this hack from this recipe.